Sources/FoundationEssentials/FileManager/FileManager+Files.swift

```diff
@@ -581,10 +581,18 @@ extension _FileManagerImpl {
             let referenceCount = UInt64(info.nNumberOfLinks)
 
             let isReadOnly = info.dwFileAttributes & FILE_ATTRIBUTE_READONLY != 0
-            var posixPermissions = UInt16(_S_IEXEC | _S_IREAD)
+            let isExecutable = if fatType == .typeDirectory {
+                true // Directories are always considered executable
+            } else {
+                SaferiIsExecutableFileType(pwszPath, 0)
```

Comment:
```suggestion
                SaferiIsExecutableFileType(pwszPath, false)
```

The second parameter is a boolean, might as well as match that :)